<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="backgroundLabel">
   <widget class="QPushButton" name="findShortestPathButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>查找最短路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setDistanceButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设置距离</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomInButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>放大地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>缩小地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveMapButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>移动地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showPointInfoButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>显示某点提示信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startPointInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>起点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endPointInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>终点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="distanceInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入距离</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="point2Input_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入景点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="point2Input">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>629</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>景点一</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="point1Input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>629</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>景点二</string>
    </property>
   </widget>
   <widget class="MapWidget" name="mapWidget">
    <property name="geometry">
     <rect>
      <x>-12</x>
      <y>-2</y>
      <width>761</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
